parcel.go

```diff
@@ -14,47 +15,118 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
 	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
+	res, err := s.db.Exec("INSERT INTO parcel (client, status, address, created_at) VALUES (?, ?, ?, ?)", p.Client, p.Status, p.Address, p.CreatedAt)
+	if err != nil {
+		return 0, err
+	}
 	// верните идентификатор последней добавленной записи
-	return 0, nil
+	number, err := res.LastInsertId()
+	if err != nil {
+		return 0, err
+	}
+	return int(number), nil
 }
 
 func (s ParcelStore) Get(number int) (Parcel, error) {
 	// реализуйте чтение строки по заданному number
 	// здесь из таблицы должна вернуться только одна строка
-
+	query := "SELECT client, status, address, created_at FROM parcel WHERE number = ?"
+	row := s.db.QueryRow(query, number)
 	// заполните объект Parcel данными из таблицы
 	p := Parcel{}
-
+	err := row.Scan(&p.Client, &p.Status, &p.Address, &p.CreatedAt)
+	if err != nil {
+		if err == sql.ErrNoRows {
```

Comment:
Лучше так сравнивать
```suggestion
		if errors.Is(err, sql.ErrNoRows) {
```